internal/autoconfig/stream_manager.go

```diff
@@ -329,17 +323,11 @@ func (s *StreamManager) dispatchEnvAction(id config.EnvironmentID, rep envfactor
 		return
 	case ActionInsert:
 		params := rep.ToParams()
```

Comment:
No more need to modify the data model before passing it deeper into the handlers.